apps/web-portal/lib/middlewares.mjs

```diff
@@ -26,3 +28,19 @@ export const urlencoded = bodyParser.urlencoded({extended: true})
 // TODO: Provide origin configuration for an implemented auth
 // label: enhancement
 export const corsMiddleware = cors({origin: '*'}) // Allow any origin because NOT USE cookies and basic auth
+
+export const expressPino = function (logger) {
+  return ExpressPino({
+    logger,
+    serializers: {
+      req: (req) => ({
+        method: req.method,
+        url: req.url,
+        traceID: req.headers['x-trace-id'] || uuid(),
```

Comment:
Is it case agnostic ? because I see that in axios client it's `X-Trace-ID` but here `x-trace-id` ?